website/_includes/head.html

```diff
@@ -31,6 +31,13 @@
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.0.0/styles/docco.min.css">
   <link rel="canonical" href="{% include canonical-link.html %}">
   <link rel="alternate" type="application/rss+xml" title="{{ site.title }}" href="{{ "/feed.xml" | prepend: site.baseurl | prepend: site.url }}">
+  <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
+  <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
+  <link rel="manifest" href="/site.webmanifest">
+  <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#36b6e5">
+  <meta name="msapplication-TileColor" content="#2d89ef">
```

Comment:
Please add icon here to after adding [all tiles](https://github.com/DevExpress/testcafe-studio-docs/pull/102/files#r223271121)
<meta name="msapplication-TileImage" content="/mstile-144x144.png">